examples/hybrid_jobs/6_QNSPSA_optimizer_with_embedded_simulator/qnspsa_with_embedded_simulator.ipynb

```diff
@@ -0,0 +1,1360 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    !cat source_scripts/QNSPSA.py
</pre><p>No strong opinion about this but maybe we don't need to print this since the code inside is not mentioned in notebook (other than there is&nbsp;<code>step</code>&nbsp;and&nbsp;<code>step_and_cost</code>&nbsp;methods).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-braket-examples/pull/149/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>